opentok/opentok.py

```diff
@@ -700,7 +702,7 @@ def get_archive(self, archive_id):
         else:
             raise RequestError("An unexpected error occurred", response.status_code)
 
-    def get_archives(self, offset=None, count=None, session_id=None):
+    def list_archives(self, offset=None, count=None, session_id=None):
```

Comment:
This would be a breaking change. If it is important to rename this method, please leave the old method name in and mark it as deprecated.